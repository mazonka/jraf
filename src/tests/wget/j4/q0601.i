command=jw md 0 /.jraf.sys/users : login admin@server.com *