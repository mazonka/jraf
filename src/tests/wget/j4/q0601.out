OK /.jraf.sys/users : OK http://127.0.0.1:16000/jraf?
==
f82d3bb17612458c
